docs/management/debugging.md

```diff
@@ -134,13 +134,13 @@ simple as using the **bt** command:
 
     (gdb) bt
     #0  debugCommand (c=0x7ffc32005000) at debug.c:220
-    #1  0x000000010d246d63 in call (c=0x7ffc32005000) at redis.c:1163
-    #2  0x000000010d247290 in processCommand (c=0x7ffc32005000) at redis.c:1305
+    #1  0x000000010d246d63 in call (c=0x7ffc32005000) at valkey.c:1163
+    #2  0x000000010d247290 in processCommand (c=0x7ffc32005000) at valkey.c:1305
     #3  0x000000010d251660 in processInputBuffer (c=0x7ffc32005000) at networking.c:959
     #4  0x000000010d251872 in readQueryFromClient (el=0x0, fd=5, privdata=0x7fff76f1c0b0, mask=220924512) at networking.c:1021
     #5  0x000000010d243523 in aeProcessEvents (eventLoop=0x7fff6ce408d0, flags=220829559) at ae.c:352
     #6  0x000000010d24373b in aeMain (eventLoop=0x10d429ef0) at ae.c:397
-    #7  0x000000010d2494ff in main (argc=1, argv=0x10d2b2900) at redis.c:2046
+    #7  0x000000010d2494ff in main (argc=1, argv=0x10d2b2900) at valkey.c:2046
```

Comment:
redis.c is was removed in Redis 3 I think. We should get a more recent stack trace when possible.

Reply:
yes, now the filename is changed to server.c. Please do not review so far, I already add "WIP" for this PR

Reply:
sorry, I opened it just to see if it was going to conflict with my other ongoing PR and noticed this. I will wait till WIP is off to review closely!